src/app.js

```diff
@@ -247,9 +254,9 @@ class Messenger extends EventEmitter {
       attachments
     } = message;
 
-    if (message.is_echo) {
-      // Requires enabling `message_echoes` in your webhook, which is not the default
-      // https://developers.facebook.com/docs/messenger-platform/webhook-reference#setup
+    if (this.options.ignoreEcho && message.is_echo) {
+      // Echo messages are ordinary messages with `is_echo` set. They're only
+      // interesting if your bot wants to listen to other bots.
```

Comment:
nope, it turns out that there is no "echo" event. Just regular messages that happen to be echos. I expected echo events that were simple ACKs but that's not what Messenger does.